src/main/java/frc/robot/RobotContainer.java

```diff
@@ -35,6 +35,7 @@ public RobotContainer() {
                 () -> -ps4_controller1.getRawAxis(1),
                 () -> -ps4_controller1.getRawAxis(0),
                 () -> -ps4_controller1.getRawAxis(2),
+                () -> (ps4_controller1.getRawAxis(4) + 1d) / 2d,
```

Comment:
Add a comment about what exactly this does (i.e takes the R2 input, shifts it from the range [-1, 1] to [0, 1], and then passes it in as a interpolation value between the min & max percent driver speed)

Reply:
thanks